scripts/ci/batch_pw_load.sh

```
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Batch script for Playwright (E2E/Visual) and Load tests using Makefile
+# This script provides a unified interface for local, GitHub, and AWS CI environments
+
+set -e
+
+# Script configuration
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/../.." && pwd)"
+MAKEFILE_PATH="$PROJECT_ROOT/Makefile"
```

Shellcheck (0.10.0)

[warning] MAKEFILE_PATH appears unused. Verify use (or export if used externally).

(SC2034)